Cargo.toml

```diff
@@ -201,3 +203,6 @@ authors = [
 ]
 license = "Apache-2.0/MIT"
 repository = "https://github.com/3box/rust-ceramic"
+
+[patch.crates-io]
+criterion2 = { git = "https://github.com/dbcfd/criterion2.rs.git", branch = "feat/async-setup" }
```

Comment:
why do we need a fork?

Reply:
Needed the PR I put up merged and released.

Reply:
It looks like it's been released? Mind updating it when you resolve the conflicts?